### PR TITLE
fix: [L07] Missing error messages in require statements 

### PR DESCRIPTION
**Pull request by md0x**
Signed-off-by: Pablo Maldonado <pablo@umaproject.org>

**Motivation**
`Within the codebase, there are multiple require statements that lack error messages.`

**Summary**

Add missing require messages

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

**Comment by md0x** [comment]
The CI is failing with:
```
Error: ERROR processing /home/circleci/protocol/packages/core/deploy/052_deploy_votingV2.js:
@uma/core: Error: Transaction reverted: trying to deploy a contract whose code is too large
```

**Review by chrismaree — COMMENTED** [review]
these require messages were removed to save bytecode. we should leave this PR until the very end to try and save the bytecode for other implementation-specific stuff and then see how many we can add back.

one additional point to make: always keep the require messages under 32 chars in length. I can see some of the ones you added might be over this. also, ideally avoid putting a full stop at the end of the require message.

**Review by mrice32 — COMMENTED** [review]
I think it might be difficult/impossible to add these back while keeping the contract size limit in check.

However, if we decide we can't, we should make sure the reasons are clearly documented above the require statement, so any user that debugs a txn and sees the line will know why the require exists!

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 1]
```diff
@@ -1011,13 +1015,14 @@ contract VotingV2 is
      */
     function executeSpamDeletion(uint256 proposalId) external {
         require(spamDeletionProposals[proposalId].executed == false);
+
```
Comment (md0x) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:1018]:
```suggestion
```

**Review by chrismaree — APPROVED** [review]

**Review by chrismaree — APPROVED** [review]

**Pull request merged**